advanced/Scripts/piholeDebug.sh

```diff
@@ -457,7 +472,7 @@ does_ip_match_setup_vars() {
   # If it's an IPv6 address
   if [[ "${protocol}" == "6" ]]; then
     # Strip off the / (CIDR notation)
-    if [[ "${ip_address%/*}" == "${setup_vars_ip}" ]]; then
+    if [[ "${ip_address%/*}" == "${setup_vars_ip%/*}" ]]; then
```

Comment:
I assume this has no effect if ${setup_vars_ip} has no CIDR? We pushed a fix into development to stop that from happening completely. The initial fix just removed it when `gravity.sh` processed it, but a later fix then stopped it from being put into setupvars with CIDR